### DISCUSSION TITLE
Functionality idea for incentivizing holder-creator interactions, reducing creator coin price fluctuations

### DISCUSSION THREAD

**Discussion by starykolarz** [deso-protocol/dips#145] [Medium CIPs]
I propose to discuss whether it would make sense to introduce functionality (sort of a composite transaction), that would do two things:
1. transfer certain amount of CC from creator coin seller to creator coin buyer
2. transfer M amount of CLOUT (agreed price of purchased CC) from creator coin buyer to creator coin seller, where Y <= M <= Z

Why?

With current CC mechanics, when a holder sells your coin, with X CC in circulation, they get less clout (Y clout), than another investor would have to pay (Z clout) to buy the same amount of your CC at the same X CC in circulation.

Advantages I see of such direct trades:
 - incentivizes holders of creator coin to communicate with creators whose coin they want to sell, before the sale - as the creator might want to purchase own coin cheaper (M) than she/he would have to pay (Z) for same amount; or might want to avoid own CC volatility. And who knows what a direct discussion might bring? (especially that BitClout imho is and should remain first and foremost a social platform, with additional element of financial value/speculation, and not the other way around)
 - reduces creator coin volatility
 - allows investors to bet/speculate on future value of creator coin, by negotiating M during direct trades

Please write what you think about this. 

Useful or feature creep? 
Would you use it to reduce/avoid CC price fluctuations?
Would you appreciate a (significant) holder of your CC contacting you before selling?

**Comment by darian-parrish** [comment]
I think this is better handled one a CC secondary trading marketplace or smart service

**Reply by arhebbar** [reply]
I agree with Darian. Probably better implemented outside of core protocol. But, in a way, isn't this like the "Buy Now" feature. It could also have been pushed to the dApps to build. 

Also, on another note, it would be good to have a categorization of DIPs such as these that are recommended for Nodes to implement. That way, these can be given to the next Devs on a hackathon so that they know what the community wants and has voted on and can prioritize these tools / features / dApps over others.

**Reply by starykolarz** [reply]
Agree, now that smart services are known - they can be used to address this. There's only one risk I see - without support at protocol level, it will require more than 1 DeSo transactions, which means that "something" can happen in between that will cause part of the "deal" to not be executed, or executed on changed terms.

**Reply by arhebbar** [reply]
It has to be handled with some logic or setup like an ESCROW to avoid these scenarios. But, even at the protocol level, the same issue does exist unless we create a new transaction and doing that will open a can of worms. What about swapping Creator Coins for NFTs, Creator Coins for Reclouts, etc
